### PR TITLE
Remove goamz

### LINKED ISSUE #1950: Use aws-sdk-go over goamz
There are still 2 users of goamz in Flynn, test/runner and util/release.
Now that aws-sdk-go has been released goamz is unsupported by it's maintainers and lacks required features.

### PR DESCRIPTION
**Pull request by josephglanville**
Goamz is no longer supported.

Closes #1950

### CODE REVIEW

File: util/release/amis.go [hunk 1]
```diff
@@ -46,15 +47,15 @@ func amis(args *docopt.Args) {
...
-			RootDeviceType:       image.RootDeviceType,
-			RootDeviceName:       image.RootDeviceName,
+			OwnerID:              *image.OwnerID,
+			RootDeviceType:       *image.RootDeviceType,
+			RootDeviceName:       *image.RootDeviceName,
 			RootDeviceSnapshotID: snapshotID,
-			VirtualizationType:   image.VirtualizationType,
-			Hypervisor:           image.Hypervisor,
+			VirtualizationType:   *image.VirtualizationType,
+			Hypervisor:           *image.Hypervisor,
```
Comment (josephglanville) [inline util/release/amis.go]: Should I change Manifest to handle *string instead of dereferencing all of these here?

**Comment by lmars** [comment]
@josephglanville did you intentionally remove `github.com/codegangsta/cli`? It seems unrelated.

**Comment by lmars** [comment]
LGTM

**Comment by josephglanville** [comment]
@lmars I think godep did that, I will tell drop that from the commit.

**Comment by titanous** [comment]
LGTM

**Comment by lmars** [comment]
@josephglanville I've pushed a fix for the S3 upload: c427e85

LGTM

**Pull request merged**